pkg/tfgen/generate_go.go

```diff
@@ -408,6 +408,9 @@ func (g *goGenerator) emitPlainOldType(w *tools.GenWriter, pot *propertyType) {
 		} else if prop.rawdoc != "" {
 			g.emitRawDocComment(w, prop.rawdoc, "\t")
 		}
+		if prop.deprecationMessage() != "" {
+			w.Writefmtln("// Deprecated: %s", prop.deprecationMessage())
```

Comment:
Nit: Looks like this is missing a `\t` at the beginning:

<img width="999" alt="Screen Shot 2020-01-09 at 2 58 55 PM" src="https://user-images.githubusercontent.com/710598/72100464-ac270680-32f0-11ea-8461-72b56838c7a2.png">

```suggestion
			w.Writefmtln("\t// Deprecated: %s", prop.deprecationMessage())
```